Releasenotes xtc 85-93, xlt 8.3-8.5 (#262)
Xceptance/xlt-documentation: content/en/xtc/loadtesting/180-reports.md

After the load test has finished, you can create as many custom reports with adjusted report settings as you need. 

> This can be done on to the _Results_ tab, either by opening the context menu of any result set and selecting [_Create Report_]({{< relref "175-results#creating-a-report-from-results" >}}), or by clicking the **NEW** button on top of the _Reports_ section. In the latter case, the report will be created from the latest available result. When determining the latest result, XTC skips pending or failed results as these typically don’t have result data attached (yet).

Reviewer: ```suggestion
This can be done on the _Results_ tab, either by opening the context menu of any result set and selecting [_Create Report_]({{< relref "175-results#creating-a-report-from-results" >}}), or by clicking the **NEW** button at the top of the _Reports_ section. In the latter case, the report will be created from the latest available result. When determining the latest result, XTC skips pending or failed results as these typically don’t have result data attached (yet).
```

In case you accidentally deleted some test report, XTC allows you to **restore** it within 30 days after deletion. Click the _Show Deleted Items_ entry in the 3-dot reports table menu to list the items that can be restored. To restore a particular item, click _Undo Deletion_ in the context menu of that item. To switch the view back to the live items, click _Hide Deleted Items_ in the table menu. Please note that load test reports that were publicly shared must be re-shared after restoring.

> ## Report Retiring

Reviewer: ```suggestion
## Automatic Report Retirement
```

How about making this section the last one on this page?

> XTC stores all these reports in two forms:

Reviewer: ```suggestion
XTC stores all load test reports and performance comparison reports in two forms:
```
